doc/api/report.md

```diff
@@ -22,6 +22,7 @@ is provided below for reference.
 ```json
 {
   "header": {
+    "reportVersion": "1.0.0",
```

Comment:
Unless we need `major.minor.patch` or `major.minor` for some reason, `"reportVersion": 1` would be sufficient.  How would minor/patch versions help here?

Reply:
I personally like the idea of a single number, but it was brought up in the original issue. Easy to change it once we figure out what we want to do.

Reply:
I don't have a lot of skin in this game, opinions of consumers (like @boneskull ) should weigh more than mine. That said, minor would be additions, like new sections, new keys, etc. major would be backwards incompatible changes. and patch... not sure, can there be a bug fix? I guess, maybe, in theory. That NAPI and ABI are versioned with single numbers though makes me think maybe this should do likewise.

Reply:
+1 for consistency with N-API and ABI versioning.

Reply:
It would be nice if some guidelines were written down as to when to bump the version and when not to bump the version. Some of the keys in the sections are obtained by looping through structures and could therefore have keys added to the report as a result of changes elsewhere in the codebase without any of the report code changing.